@LinProp/unc_budget.m

```diff
@@ -1,7 +1,30 @@
-% LinProp Uncertainty Budget
-% Michael Wollensack METAS - 20.09.2021
-
 function unc_budget(x, varargin)
+% UNC_BUDGET Opens budget window.
+%
+% unc_budget(unc) opens a window with the uncertainty budget of unc.
+% The input unc must be a LinProp. It can be a scalar, vector, or matrix.
+%
+% unc_budget(unc, format) behaves as above, but also specifies the format
+% of numerical values. format must be a char array or string scalar. 
+% Recomended values are 'f3', 'f6', 'f9', 'e3', 'e6', and 'e9', but
+% <a href="https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings">standard</a> and <a
+% href="https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings">custom</a> .NET numeric format strings will also work.
+%
+% unc_budget(unc, format, windowName) behaves as above, but also specifies
+% the name of the budget window. windowName must be a string scalar or char
+% array. The default value is the name of the variable passed as unc.
+%
+% unc_budget(unc, format, windowName, columnNames) behaves as above, but
+% also specifies the names of the elements of unc and thus the column
+% headings in the GUI. columnNames must be a cell string or array of
+% strings. This option is only available when var is complex or a vector 
+% or matrix.
+%
+% If unc contains a complex values vector, the real and imag part are
+% interleaved. In this respect unc_budget behaves differently than
+% get_correlation(), get_covariance(), etc.
```

Comment:
I cannot see the different behavior of `unc_budget` here:

Example:

```
>> a = LinProp(3,0.3)

a =

   (3 ± 0.3000)

>> b = LinProp(4,0.4)

b =

   (4 ± 0.4000)

>> c = LinProp(5,0.5)

c =

   (5 ± 0.5000)

>> d = LinProp(6,0.6)

d =

   (6 ± 0.6000)

>> x = [a+1i*b  c+1i*d]
 
x.Value = 
 
   3.0000 + 4.0000i   5.0000 + 6.0000i

 
x.StdUnc = 
 
   0.3000 + 0.4000i   0.5000 + 0.6000i

 
Did you mean:
>> get_covariance(x)

ans =

    0.0900         0         0         0
         0    0.1600         0         0
         0         0    0.2500         0
         0         0         0    0.3600
```

and the `unc_budget(x)`:

![image](https://user-images.githubusercontent.com/22192118/202458057-d93b9f50-5271-46d8-ae1c-cfbd96fa62de.png)

Reply:
Good that you noticed that! The issue here was with the documentation of several other methods.

The documentation of `get_covariance` for example contained the following statement:
> The input argument unc is always interpreted as a vector, thus get_covariance(unc) is the same as get_covariance(unc(:)). If unc contains complex values, the real and imag part are treated separately, thus get_covariance(cUnc) is the same as get_covariance([real(cUnc(:)), imag(cUnc(:))]).

The first sentence is true. The second sentence, about real and imag, however is not - it is missing a `'` at the end and should read:
> The input argument unc is always interpreted as a vector, thus get_covariance(unc) is the same as get_covariance(unc(:)). If unc contains complex values, the real and imag part are treated separately, thus get_covariance(cUnc) is the same as get_covariance([real(cUnc(:)), imag(cUnc(:))]').

This mistake in the documentation lead me to believe there would be a difference in the behaviour of the two functions. I have now updated all affected parts of the documentation to also explain what will happen using words:
> The input argument unc is always interpreted as a vector, thus get_covariance(unc) is the same as get_covariance(unc(:)). If unc contains complex values, the real and imag part are interleaved with each imag part right after its respective real part. This is the same as passing [real(unc(:)), imag(unc(:))]' instead of unc.

Reply:
I saw two typos:

> the real and imag part are interleaved with each imag part right after its respective real part.

the real and imag**inary** part**s** are interleaved with each imag**inary** part right after its respective real part.

After fixing that I think we are finally ready to merge this pull request.

Reply:
I have fixed the typos you mentioned.